libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1559,6 +1578,19 @@ void AP_OSD_Screen::draw_clk(uint8_t x, uint8_t y)
     }
 }
 
+void AP_OSD_Screen::draw_rngf(uint8_t x, uint8_t y)
+{
+    RangeFinder *rangefinder = RangeFinder::get_singleton();
+    AP_RangeFinder_Backend *sensor = rangefinder->get_backend(0);
+    if (sensor == nullptr) {
+        backend->write(x, y, false, "RNGF: NO SENSOR");
+    } else if  (!sensor->has_data()) {
```

Comment:
if (rangefinder.status_orient(ROTATION_PITCH_270) != RangeFinder::Status::NoData) {

Reply:
This will remove the dependency on the AP_RangeFinder_Backend and get the right rangefinder that is assume to be the altitude one's